kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/model/VirtualCluster.java

```diff
@@ -229,6 +256,33 @@ private static NettyTrustProvider configureTrustProvider(Tls tlsConfiguration) {
         return new NettyTrustProvider(trustProvider);
     }
 
+    private static void configureCipherSuites(SslContextBuilder sslContextBuilder, Tls tlsConfiguration) {
+        Optional.ofNullable(tlsConfiguration.cipherSuites())
+                .map(ciphers -> sslContextBuilder.ciphers(
+                        tlsConfiguration.cipherSuites().allowed(),
+                        new DenyCipherSuiteFilter(tlsConfiguration.cipherSuites().denied())));
+    }
+
+    private static void configureEnabledProtocols(SslContextBuilder sslContextBuilder, Tls tlsConfiguration) {
+        var protocols = Optional.ofNullable(tlsConfiguration.protocols());
+
+        if (protocols.isPresent()) {
+            var allowedProtocols = Optional.ofNullable(protocols.get().allowed()).orElse(Set.of(Protocols.TLSv1_2, Protocols.TLSv1_3));
```

Comment:
I'm in the middle of getting a Platforms supported by Default protocols to fill this by the way, but thought best to commit in the rest of changes in case other comments.